releases.yaml

```diff
@@ -19,17 +19,23 @@ previous_lts:
   name: "Bionic Beaver"
   short_version: "18.04"
   full_version: "18.04.5"
+  release_date: "April 2018"
+  eol: "April 2023"
 previous_previous_lts:
   name: "Xenial Xerus"
   short_version: "16.04"
   full_version: "16.04.7"
 
 checksums:
   desktop:
+    "20.10": "b45165ed3cd437b9ffad02a2aad22a4ddc69162470e2622982889ce5826f6e3d *ubuntu-20.10-desktop-amd64.iso"
```

Comment:
Should we leave these empty or just remember to update these?

Reply:
I will create an issue now